pages/owned/index.js

```diff
@@ -26,5 +31,14 @@ export default function Owned() {
       </p>
     );
   }
-  return <PlantList plants={ownedData} />;
+
+  const filteredPlantList = filteredPlants ?? ownedData;
```

Comment:
make sure that you filter the ownedData here instead of data when applying the other filter approach.